### DISCUSSION TITLE
Verbindung Hersteller/Produkt/Impfstoff

### DISCUSSION THREAD

**Discussion by pbroska** [Digitaler-Impfnachweis/certification-apis#154] [Q&A]
Wo steht die Verbindung zwischen Hersteller / Produkt / Impfstoff und
Gesamtdosen? Die Links aus der Spezifikation "eHealth Network"
(https://ec.europa.eu/health/sites/default/files/ehealth/docs/digital-green-certificates_dt-specifications_en.pdf)
auf den Seiten der EU
("https://ec.europa.eu/dgc/DGC.ValueSets.schema.json#/$defs/disea
se-agent-targeted et al.) sind alle nicht mehr gültig, über die
EU-Suchmaschine ist nichts zu finden

**Comment by oliver-steinbrecher** [comment]
@thinkberg Könnt ihr eure Mapping Tabelle vom Web-Frontend auch im Github veröffentlichen?
